accounts/pkg/flagset/flagset.go

```diff
@@ -99,6 +99,62 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVars:     []string{"ACCOUNTS_JWT_SECRET"},
 			Destination: &cfg.TokenManager.JWTSecret,
 		},
+		&cli.StringFlag{
+			Name:        "storage-disk-path",
+			Value:       "",
```

Comment:
can we have a default value?

Reply:
No. ;-) We have default values for the CS3 `Repo` implementation, since it's supposed to be the default. When any other storage is configured (we only have the disk implementation as "other storage") we switch over to that.

TL;DR: Setting a default here would switch back to using the local disk for persisting accounts. We want to use CS3 as default, so we can't have a default disk path.